### PR TITLE
chore: add new dashboard/alerts info events

### PR DESCRIPTION
**Pull request by makeavish**
### Summary

Follow this doc for info : https://www.notion.so/signoz/Alert-Dashboard-events-efb823a9d874438aae3ffb4e738ee31b?pvs=4

- New Event - Dashboard/Alert Info
- Update heartbeat interval to 12 hrs and active user interval to 6hrs
- Remove all APIs from telemetry from OSS apart from` api/v1/channels`

### CODE REVIEW

File: pkg/query-service/app/clickhouseReader/reader.go [hunk 1]
```diff
@@ -3421,6 +3423,100 @@ func (r *ClickHouseReader) GetTagsInfoInLastHeartBeatInterval(ctx context.Contex
...
+			alertsInfo.MetricBasedAlerts = alertsInfo.MetricBasedAlerts + 1
+		} else if rule.AlertType == "TRACES_BASED_ALERT" {
+			alertsInfo.TracesBasedAlerts = alertsInfo.TracesBasedAlerts + 1
+		}
+		alertsInfo.TotalAlerts = alertsInfo.TotalAlerts + 1
+	}
+
+	return &alertsInfo, nil
+}
+
```
Comment (srikanthccv) [inline pkg/query-service/app/clickhouseReader/reader.go:3519]: None of this belongs in the ClickHouse reader. There is nothing ClickHouse related here. Please move elsewhere.

File: pkg/query-service/interfaces/interface.go [hunk 2]
```diff
@@ -71,6 +71,8 @@ type Reader interface {
 	GetListResultV3(ctx context.Context, query string) ([]*v3.Row, error)
 	LiveTailLogsV3(ctx context.Context, query string, timestampStart uint64, idStart string, client *v3.LogsLiveTailClient)
 
+	GetDashboardsInfo(ctx context.Context) (*model.DashboardsInfo, error)
+	GetAlertsInfo(ctx context.Context) (*model.AlertsInfo, error)
```
Comment (srikanthccv) [inline pkg/query-service/interfaces/interface.go:75]: Reader interface shouldn't be bothered about this unrelated telemetry detail.

Comment (makeavish) [reply pkg/query-service/interfaces/interface.go:75]: Similar telemetry functions are already part of reader so this will require major refactoring

Comment (makeavish) [reply pkg/query-service/interfaces/interface.go:75]: Can take it up separately: https://github.com/SigNoz/engineering-pod/issues/1141

File: pkg/query-service/app/clickhouseReader/reader.go (hunk 1 above)
Comment (makeavish) [reply pkg/query-service/app/clickhouseReader/reader.go:3519]: Can take it up separately: https://github.com/SigNoz/engineering-pod/issues/1141

File: pkg/query-service/telemetry/telemetry.go [hunk 3]
```diff
@@ -61,9 +63,9 @@ const ph_api_key = "H-htDCae7CR3RV57gUzmol6IAKtm5IMCvbcm_fwnL-w"
 const IP_NOT_FOUND_PLACEHOLDER = "NA"
 const DEFAULT_NUMBER_OF_SERVICES = 6
 
-const HEART_BEAT_DURATION = 6 * time.Hour
+const HEART_BEAT_DURATION = 12 * time.Second
```
[the remark this answers is not in the document]
Comment (makeavish) [reply pkg/query-service/telemetry/telemetry.go:66]: Should be hours, accidentally pushed testing version

File: pkg/query-service/telemetry/telemetry.go [hunk 4]
```diff
@@ -61,9 +63,9 @@ const ph_api_key = "H-htDCae7CR3RV57gUzmol6IAKtm5IMCvbcm_fwnL-w"
...
 
-const ACTIVE_USER_DURATION = 30 * time.Minute
+const ACTIVE_USER_DURATION = 6 * time.Second
```
[the remark this answers is not in the document]
Comment (makeavish) [reply pkg/query-service/telemetry/telemetry.go:68]: Should be hours, accidentally pushed testing version

File: pkg/query-service/telemetry/telemetry.go [hunk 5]
```diff
@@ -241,9 +243,27 @@ func createTelemetry() {
 				}
 				telemetry.SendEvent(TELEMETRY_EVENT_HEART_BEAT, data, "")
 
+				alertsInfo, err := telemetry.reader.GetAlertsInfo(context.Background())
+				dashboardsInfo, err := telemetry.reader.GetDashboardsInfo(context.Background())
+
+				dashboardsAlertsData := map[string]interface{}{
+					"totalDashboards":   dashboardsInfo.TotalDashboards,
```
[the remark this answers is not in the document]
Comment (makeavish) [reply pkg/query-service/telemetry/telemetry.go]: value might be nil, returning non nil dashboard and also moved the struct on condition which checks for error

**Review by ankitnayan — APPROVED** [review]

File: pkg/query-service/telemetry/telemetry.go [hunk 6]
```diff
@@ -241,9 +243,27 @@ func createTelemetry() {
 				}
 				telemetry.SendEvent(TELEMETRY_EVENT_HEART_BEAT, data, "")
 
+				alertsInfo, err := telemetry.reader.GetAlertsInfo(context.Background())
```
[the remark this answers is not in the document]
Comment (makeavish) [reply pkg/query-service/telemetry/telemetry.go:246]: Yes though it is non serious, will update in separate PR.

**Pull request merged**